src/mlpack/methods/reinforcement_learning/replay/random_replay.hpp

```diff
@@ -186,7 +186,7 @@ class RandomReplay
               arma::irowvec& isTerminal)
   {
     size_t upperBound = full ? capacity : position;
-    arma::uvec sampledIndices = arma::randi<arma::uvec>(
+    arma::uvec sampledIndices = randi<arma::uvec>(
         batchSize, arma::distr_param(0, upperBound - 1));
```

Comment:
This is a comment for later, but I'm interested what will happen here when we remove the `arma::` from `distr_param`.  If we have Bandicoot available, then both the `arma::distr_param` and `coot::distr_param` classes will be available, and the compiler will have to correctly select `arma::distr_param`.  I *think* that this will work fine when we do it, but there is a possibility that there could be a problem we need to work around.

Anyway, nothing to do about that now, just a comment.

Reply:
I know, when I had ADL I was planning to use an additional template parameter `DistrParam`. However, there is a high chance that we will not needed for now.